src/pls/table.py

```diff
@@ -52,7 +73,10 @@ def get_columns() -> list[str]:
 
     cols = []
     if args.details:
-        cols.extend(["type", "perms", "", "user", "group", "", "size", ""])
+        if platform != "win32":
+            cols.extend(["type", "perms", "", "user", "group", "", "size", ""])
+        else:
+            cols.extend(["type", "perms", "", "", "size", ""])
```

Comment:
Remove two consecutive spacers.
```suggestion
            cols.extend(["type", "perms", "", "size", ""])
```

Reply:
Yep, that's what I was originally going to do. I just wasn't sure, since there were dummy columns in column_spec, so I didn't know if they were needed or not.